app/views/staff/activities/children.html.haml

```diff
@@ -0,0 +1,46 @@
+= content_for :page_title_prefix, t("document_title.activity.children", name: @activity.title)
+
+%main.govuk-main-wrapper#main-content{ role: "main" }
+  .govuk-grid-row
+    .govuk-grid-column-full
+      %h1.govuk-heading-xl
+        = @activity.title
+
+  .govuk-grid-row
+    .govuk-grid-column-full
+      - if @activity.project? || @activity.third_party_project?
+        - if policy(:project).download? || policy(:third_party_project).download?
+          = link_to t("default.button.download_as_xml"), organisation_activity_path(@activity.organisation, @activity, format: :xml), class: "govuk-button"
+
+  .govuk-grid-row
+    .govuk-grid-column-full
+      .govuk-tabs
+        %h2.govuk-tabs__title
+          = t("tabs.activity.title")
+
+        %ul.govuk-tabs__list
+          %li.govuk-tabs__list-item
+            = link_to t("tabs.activity.financials"),
+              organisation_activity_financials_path(@activity.organisation, @activity),
+              { class: "govuk-tabs__tab", role: "tab", tabindex: 0, aria: { controls: "financials", selected: true } }
+          %li.govuk-tabs__list-item
+            = link_to t("tabs.activity.details"),
+              organisation_activity_details_path(@activity.organisation, @activity),
+              { class: "govuk-tabs__tab", role: "tab", tabindex: 0, aria: { controls: "details", selected: true } }
```

Comment:
It's a really good point, thanks! I've had a look and I must've been viewing the rendered markup in the browser inspector when I did this as the actual source on GOVUK has no tabindex! Great spot! I'll get rid of them and let the browser do it's thing as you propose.

Reply:
Okay, dug deeper. The attributes are missing on the markup example because the expected way to use tabs is with JS, so they add all the attributes via JS to present the tabs (and fallback to a flat list).

As we don't use the JS we should follow the rendered markup which incudes the `tabindex` and all inactive tabs are `-1` and the active tab is `0` so I have it the wrong way – I'll fix it!

Reply:
This might now be an aside but I thought we did have the govuk-frontend package installed and running. Is that a different thing? 

If `-1` is to prevent the link from being focused at all and `0` is to provide focus. I wonder if we can/should omit it when it's `-1`? Looking at our top nav bar we don't set any tab indexes and they seem to be ignored. Equally that could be unconventional and we should set about slowly changing things to set the attribute explicitly?

Reply:
I am wary of the rabbit hole here! I've looked at how the GOVUK JS ones work, (in Chrome) you tab to the current tab and then user the left and right arrow keys to move tab, this must be JS powered.

We can't fire the GOVUK tab JS on ours as each of our tabs is a seperate page not content hidden and shown via JS., they're JS just errrors.

If we remove the `tabindex` ours works by tabbing to each tab which I think is a good enough compromise, let me know what you think?

Reply:
I think that's fine. 

It looks like their JS is toggling the tabindex field to `0` and back to `-1` depending on which is the active tab. Given we are doing new page loads on every click our focus resets which isn't excellent for users who want to use tabs to navigate. I think it's good enough for now, if it comes up in our next accesibility audit we can do something to dynamically set the tabindex based on active page.